src/reconcile/rec_visibility.c

```diff
@@ -281,6 +282,18 @@ __wt_rec_upd_select(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_INSERT *ins, v
         if (upd->start_ts > max_ts)
             max_ts = upd->start_ts;
 
+        /*
+         * FIXME-PM-1524: A temporary solution for not storing durable timestamp in the cell.
```

Comment:
The PM project in JIRA is private, and our source is public. It's better not to reference it directly. Maybe replace PM-1524 in the comment with a WT JIRA ticket to remove this block of code (which can be linked to PM-1524)

Reply:
It has become the common practice to use the project ticket in durable history. I'm not sure we should change this as it will become more difficult for us to search these places if we mark them inconsistently. @bvpvamsikrishna What's your idea?

Reply:
Yes, we will not use PM-1524, instead use prepare support.